src/pow.rs

```diff
@@ -1,6 +1,81 @@
 use core::ops::Mul;
 use {One, CheckedMul};
 
+/// Binary operator for raising a value to a power.
+pub trait Pow<RHS> {
+    /// The result after applying the operator.
+    type Output;
+
+    /// Returns `self` to the power `rhs`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num_traits::Pow;
+    /// assert_eq!(10.pow(2), 100);
```

Comment:
CI failed here because it inferred this like `10i32.pow(2i32)`, which is not implemented.